initiativeKeeper.py

```diff
@@ -17,13 +18,16 @@ def runCommand(self,message):
             return self.printInitiatives(guild_id,message.guild.name)
         elif command.startswith('add'):
             return self.addInitiative(guild_id, command.replace('add ',''))
+        elif command.startswith('next'):
+            return self.nextInitiative(guild_id)
         else:
             return 'Invalid command'
 
     def addInitiative(self,guild_id,command):
         try:
             char,init = command.split(' ')
             self.guild_initiatives[guild_id][char] = init
+            self.init_index = 0
```

Comment:
The init index must be relative to the guild_id.The bot shares one instance across many guilds and channels. As is right now, modifying the init_index property in one server will modify it for all servers